tests/Integration/Behaviour/Features/Scenario/Title/title_management.feature

```diff
@@ -6,26 +6,26 @@ Feature: Title management
 
   Background:
     Given I define an uncreated title "titleNotFound"
+    And language "en" with locale "en-US" exists
+    And language "fr" with locale "fr-FR" exists
 
   Scenario: Adding new title
     When I add a new title "3rdTitle" with the following properties:
-      | name    | Third Title    |
-      | type    | Other          |
-#    Then title "thirdTitle" name should be "Normandy"
-#    Then title "thirdTitle" country should be "United titles"
-#    Then title "thirdTitle" zone should be "Europe"
-#    And title "thirdTitle" should be enabled
-#
-#  Scenario: Editing title
-#    When I edit title "thirdTitle" with following properties:
-#      | name    | Britain       |
-#      | enabled | false         |
-#      | country | Italy         |
-#      | zone    | South America |
-#    Then title "thirdTitle" name should be "Britain"
-#    Then title "thirdTitle" country should be "Italy"
-#    Then title "thirdTitle" zone should be "South America"
-#    And title "thirdTitle" should be disabled
+      | name[en-US] | Third Title EN |
+      | name[fr-FR] | Third Title FR |
+      | type        | Other          |
+    Then the title "3rdTitle" name should be "Third Title EN" in locale "en-US"
+    Then the title "3rdTitle" name should be "Third Title FR" in locale "fr-FR"
+    And the title "3rdTitle" gender type should be "Other"
+
+  Scenario: Editing title
+    When I edit title "3rdTitle" with following properties:
+      | name[en-US] | Fourth Title EN |
+      | name[fr-FR] | Fourth Title FR |
+      | type        | Female          |
+    Then the title "3rdTitle" name should be "Fourth Title EN" in locale "en-US"
+    Then the title "3rdTitle" name should be "Fourth Title FR" in locale "fr-FR"
+    And the title "3rdTitle" gender type should be "Female"
```

Comment:
As I suggested in the context class this should all be tested in a single step with data in table like this:

```suggestion
    Then the title "3rdTitle" should have the following properties:
      | name[en-US] | Fourth Title EN |
      | name[fr-FR] | Fourth Title FR |
      | type        | Female          |
```

Reply:
Fixed